JASP-Engine/JASP/R/anovarepeatedmeasures.R

```diff
@@ -560,33 +227,46 @@ AnovaRepeatedMeasures <- function(dataset=NULL, options, perform="run", callback
   
   main <- paste("(",paste(unlist(terms.base64), collapse=" + "),")", sep="")
   termsBS <- paste("(",paste(termsBS.base64, collapse=" + "),")", sep="")
-  errorRM <- paste("Error(",paste("subject/(", termsRM.base64, ")",sep="", collapse=" + "),")",sep="")
+  errorRM <- paste("Error(",paste("Xc3ViamVjdA/(", termsRM.base64, ")",sep="", collapse=" + "),")",sep="")
```

Comment:
We're actually moving away from base64 encoding, so this will break once you rebase. If you use `subject` and `dependent` directly, instead of their base64 equivalents, then you should be fine. 

The new encoding only encodes/decodes the actual column names. So calling `.v("dependent")` will return `"dependent"` which is perfect here